k8s/helm/yona/values.yaml

```diff
@@ -184,15 +184,21 @@ apple:
   ca_certificate_file: apple/AppleWWDRCA.cer
 
 mariadb:
-  mariadbRootPassword: develop
-  mariadbUser: develop
-  mariadbPassword: develop
-  mariadbDatabase: yona
+  db:
+    user: develop
+    password: develop
+    name: yona
+  rootUser:
+    password: develop
   persistence:
-    storageClass: standard
-    size: 2Gi
+    enabled: false
+  replication:
+    enabled: false
+  volumePermissions:
+    enabled: true
   user: develop
   password: develop
+  mariadbDatabase: yona
```

Comment:
This one is now obsolete, right?

Reply:
Technically no, since this one is used for the deployments and the other tells the mariadb chart what database to create (unused in environments with already existing mysql servers), but I don't see why we can't unify them to be the same thing like it was before (the newer mysql chart changed). I'll update and push.

Reply:
I see. I misinterpreted what caused the change. It's indeed good to unify it again.